zuul.d/adoption.yaml

```diff
@@ -127,6 +127,7 @@
       - ^roles/openshift_setup/.*
       - ^roles/repo_setup/.*
       - ^roles/cifmw_setup/.*
+      - ^roles/adoption_osp_deploy/.*
```

Comment:
this role is not used by adoption-standalone job, so it doesn't make sense to include it in files

Reply:
ah, right.